src/parseJsonReports.ts

```diff
@@ -5,30 +5,38 @@ import { JsonFinal } from './types/JsonFinal';
 import { JsonSummary } from './types/JsonSummary';
 import { stripIndent } from 'common-tags';
 
-const parseJsonReport = async <type extends JsonSummary | JsonFinal>(jsonSummaryPath: string): Promise<type> => {
-  const resolvedJsonSummaryPath = path.resolve(process.cwd(), jsonSummaryPath);
+/** Parse a Vitest coverage reporter file */
+const parseJsonReport = async <type extends JsonSummary | JsonFinal>(jsonPath: string): Promise<type> => {
+  // Input path may already be an absolute path (will shortcut 'path.resolve' call).
+  const resolvedJsonSummaryPath = path.resolve(process.cwd(), jsonPath);
   const jsonSummaryRaw = await readFile(resolvedJsonSummaryPath);
   return JSON.parse(jsonSummaryRaw.toString()) as type;
 }
 
+/**
+ * Parse Vitest coverage 'json-summary' reporter file
+ *
+ * @throws Errors when parsing fails
+ */
```

Comment:
Same as above - if you think the function name is not clear enough, let's try to improve that one before adding comments. 🙂

Reply:
This is even more true as the `json-summary` reporter file name can obviously be customized 🤦. The one thing I am in favour of function docstrings is the `@throws` annotations, as there is no way to indicate this via TS (that I am aware of); however, it is a relatively small thing. If I had to guess, I most likely added this annotation, and then a summary (since docstring was there then), and then added docstrings to the other functions here 🤷?